scenarios/kubernetes_build.py

```diff
@@ -71,6 +102,13 @@ def main(args):
     else:
         check('make', 'release')
     check('../release/push-build.sh', *push_build_args)
+    artifacts = '%s/_artifacts' % os.environ['WORKSPACE']
```

Comment:
this line came from some of the other scripts, we have a lot of logic that assumes this magic `_artifacts` dir.